cmu_graphics/cmu_graphics.py

```diff
@@ -700,6 +720,8 @@ def run(self):
                         self._height = event.h
                         self.onResize(False)
 
+                    pygameEvent.send_robust(event, callUserFn=self.callUserFn, app=self._wrapper)
+
                 should_redraw = had_event
 
```

Comment:
Were we also going to add an "as often as possible" event around here?

Reply:
Pass in msPassed to this event maybe.